harvard-huit/Harvard-Geospatial-Library
# Remove unused download section and supress download link for Harvard …

…items

**The title of this pull-request should be a brief description of what the pull-request fixes/improves/changes. Ideally 50 characters or less.**
* * *

**JIRA Ticket**: (link)

* Other Relevant Links (Mailing list discussion, related pull requests, etc.)

# What does this Pull Request do?
We don't want the individual download link on Harvard files. This removes that option from Harvard files, but leaves it for others.

# How should this be tested?

Rebuild local after pulling this branch and go to a Harvard item. You shouldn't see the export section. Go to another item from another school, and you should see it.

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? N
- integration tests? N
